sdk/servicebus/Azure.Messaging.ServiceBus/tests/Receiver/ReceiverLiveTests.cs

```diff
@@ -597,7 +597,7 @@ await using (var scope = await ServiceBusScope.CreateWithQueue(enablePartitionin
 
                 await receiver.RenewMessageLockAsync(receivedMessage);
 
-                Assert.True(receivedMessage.LockedUntil >= firstLockedUntilUtcTime + TimeSpan.FromSeconds(10));
+                Assert.True(receivedMessage.LockedUntil >= firstLockedUntilUtcTime);
```

Comment:
Should this just be greater than, or are we good with the value not changing at all after the delay?